Azrux/Athenas-front
# Created home page

## Description
* Home Page created
* Fixed styles in layout component and children components
* Moved non common components to src/components folder
* Added images for the carousel
* Created temporal json to show products
* Removed test component

## Screenshots
Desktop
![image](https://github.com/Azrux/Athenas-front/assets/89431511/2f16e242-08ff-4fdc-bc6d-c64af9a9e008)

Mobile
![image](https://github.com/Azrux/Athenas-front/assets/89431511/fa4ed6fa-6ffe-4056-8f83-f076881045b8)

## How to test?
* Run `npm run dev`
* Go to: http://localhost:3000/home
* Check everything is working fine
* Check the responsive design
